UI/EmuScreen.cpp

```diff
@@ -180,6 +180,7 @@ void EmuScreen::dialogFinished(const Screen *dialog, DialogResult result) {
 		quit_ = false;
 	}
 	RecreateViews();
+	gpu->CheckGPUFeatures();
```

Comment:
Wouldn't this go better e.g. one of these places where we do similar things?

https://github.com/hrydgard/ppsspp/blob/gpu-feature-cleanup/GPU/GLES/GLES_GPU.cpp#L683
https://github.com/hrydgard/ppsspp/blob/gpu-feature-cleanup/GPU/GLES/GLES_GPU.cpp#L2277

Other places send signals that call Resized when changing settings.  I feel like it's better to centralize.

-[Unknown]